demos/common/python/openvino/model_zoo/model_api/README.md

```diff
@@ -0,0 +1,150 @@
+# Python* Model API package
+
+Model API package is a set of wrapper classes for particular tasks and model architectures, simplifying data preprocess and postprocess as well as routine procedures (model loading, asynchronous execution, etc...)
+Model API wrappers hide the specific code inside and work as black-box: application feeds model class with input data, then the model returns post-processed output data in user-friendly format.
+
+## Package structure
+
+The Python* Model API consists of 3 libraries:
+* _adapters_ - implement a common interface to allow Model API wrappers usage with different executors: OpenVINO, ONNX, etc. See [Model API adapters](#model-api-adapters) section
+* _models_ - implement wrappers for each architecture. See [Model API Wrappers](#model-api-wrappers) section
+* _pipelines_ - implement pipelines for model inference and manage the synchronous/asynchronous execution. See [Model API Pipelines](#model-api-pipelines) section
+
+### Prerequisites
+
+The package requires
+- Python (version 3.6 or higher)
+- OpenVINO™ toolkit
```

Comment:
OpenCV is missing

Reply:
OpenCV is installed from requirements automatically